docs/dcos.md

```diff
@@ -1,5 +1,8 @@
 # Microsoft Azure Container Service Engine - DC/OS Walkthrough
 
+### `Note:`
+The support for DC/OS `1.11` and later has been continued in the forked project [dcos-engine](https://github.com/Azure/dcos-engine)

```

Comment:
I would reword this slightly and add a period at the end:

Support for DC/OS `1.11` and later continues in the forked project [dcos-engine](https://github.com/Azure/dcos-engine).